### DISCUSSION TITLE
Blank White Page After Deploying React App

### DISCUSSION THREAD

**Discussion by Tmalone1250** [community/community#140198] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I am having an issue with deploying my Final Project to GitHub Pages. When I load the URL, only a blank white page is displayed. I've added the deploy and build script as instructed as well as a homepage script into the package.json file. I've, also, updated the repository name in the base script inside of the vite.config.js file. I'm not getting any errors within the terminal, but when I inspect the blank page in Edge, I am seeing that there is an error in the console stating that the "./src/main.jsx" file cannot be found. But the file is located in the correct directory. I am also receiving a 404 error within the browser's console. I have provided a link to the repository as well as the hosted page:

- https://github.com/Tmalone1250/e-plantShopping3.git

- https://Tmalone1250.github.io/e-plantShopping3

**Comment by LiteBrite82** [comment, marked as answer]
@Tmalone1250 
It looks like you are deploying from Main; where your source code is located. You need to run a build of your source code and deploy it from the gh-pages branch. 
Vite / Pages deployment guide: https://vitejs.dev/guide/static-deploy
